### DISCUSSION TITLE
Mac M1 Intel 打包 Netflix Protonmail

### DISCUSSION THREAD

**Discussion by kxxc** [tw93/Pake#275] [Q&A] [answered]
試著用 M1 打包 Netflix Protonmail 最後還是會錯誤
改用舊 Intel Mac 嘗試 結果還是沒能成功😂

**Comment by tw93** [comment, marked as answer]
看看你的错误日志

**Reply by Tlntin** [reply]
大概率是mac平台默认打包多架构引起的问题。

**Reply by Tlntin** [reply]
新版本应该解决了，重新安装一下pake-cli试试。

**Reply by kxxc** [reply]
> 新版本应该解决了，重新安装一下pake-cli试试。

謝謝 [Tlntin](https://github.com/Tlntin)

**Comment by kxxc** [comment]
[![](https://i.imgur.com/VjHhPOY.png)](url)
[![](https://i.imgur.com/VqA3dQF.png)](url)

非常感謝 這次非常順利成功了
而且支援 4k
不過打開會有些圖片顯示不了 而且不知道怎麼 reload 😂

Disney+ 也可以打包

M1 Mac mini macOS 13.2

**Reply by tw93** [reply]
command + r 刷新

**Reply by kxxc** [reply]
> command + r 刷新

哦 可以欸 但還是有破圖問題

因為我用 BetterTouchTool
設定 F18 是 command + r 卻一直沒反應
